docs/data/base/components/button/button.md

```diff
@@ -32,14 +32,14 @@ The Button behaves similar to the native HTML `<button>`, so it wraps around the
 The following demo shows how to create and style two basic buttons.
 Notice that the second button cannot be clicked due to the `disabled` prop:
 
-{{"demo": "UnstyledButtonsSimple"}}
+{{"demo": "UnstyledButtonsSimple.js"}}
```

Comment:
It's too early, I'm removing these Tailwind CSS and CSS modules extra demos. We only have bandwidth for one, we are going to die on the while if convert everything without automation.